docs/_embed/partner_api.md

```diff
@@ -365,115 +394,131 @@ Get all Zaps in the user's account that have a particular Trello board (assuming
 curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/zaps?&params__board=BOARD_ID"
 ```
 
+Get all Zaps in the user's account and include Trello steps' `params` key/value pairs (assuming the OAuth app is Trello).
+
+```bash
+curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/zaps?&get_params"
+```
+
 **Example Response**
 
 > Refer to the [Zap](#zap) reference for all the available fields.
 
 ```json
 {
-  "next": "https://api.zapier.com/v1/zaps?limit=2&offset=2",
-  "previous": null,
-  "count": 4,
-  "objects": [
-    {
-      "id": 125,
-      "modified_at": "2017-03-22T09:38:11-05:00",
-      "state": "on",
-      "steps": [
+    "next": "https://api.zapier.com/v1/zaps?limit=2&offset=4",
+    "previous": "https://api.zapier.com/v1/zaps?limit=2",
+    "count": 20,
+    "objects": [
         {
-          "app": {
-            "description": "Typeform helps you ask awesomely online! If you ever need to run a survey, questionnaire, form, contest etc... Typeform will help you achieve it beautifully across all devices, every time, using its next generation platform.",
-            "hex_color": "8bcbca",
-            "id": 4259,
-            "uuid": " b9df4eff-f311-44f9-ac54-2901f952c6ac",
-            "image": "https://cdn.zapier.com/storage/developer/5e21b4c1e0a2a3346a801dbc0a2a5a6d_2.png",
-            "images": {
-              "url_128x128": "https://cdn.zapier.com/storage/developer/5e21b4c1e0a2a3346a801dbc0a2a5a6d_2.128x128.png",
-              "url_16x16": "https://cdn.zapier.com/storage/developer/5e21b4c1e0a2a3346a801dbc0a2a5a6d_2.16x16.png",
-              "url_32x32": "https://cdn.zapier.com/storage/developer/5e21b4c1e0a2a3346a801dbc0a2a5a6d_2.32x32.png",
-              "url_64x64": "https://cdn.zapier.com/storage/developer/5e21b4c1e0a2a3346a801dbc0a2a5a6d_2.64x64.png"
-            },
-            "api": "TypeformDevAPI",
-            "slug": "typeform",
-            "title": "Typeform",
-            "url": "https://zapier.com/apps/typeform/integrations"
-          },
-          "type_of": "read"
+            "id": 159061877,
+            "title": "",
```

Comment:
Is a blank `title` expected?

Reply:
@gregilo `/v1/zaps` is calls the `/v4/zaps/` endpoint. And AFAICT `/v4/zaps/` allowed blanks strings so it is to be expected.

Reply:
@MagWeaver gotcha - thanks for looking into this further!